cli_pr_review.py

```diff
@@ -11,7 +11,7 @@
 app = typer.Typer()
```

Comment:
The updated spinner text "Retrieving pull request data" is more accurate and descriptive. Good change.